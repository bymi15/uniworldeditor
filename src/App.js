import React, { Fragment } from "react";
import { Redirect, BrowserRouter, Route, Switch } from "react-router-dom";
import Layout from "./components/Layout";
import Home from "./pages/Home";
import About from "./pages/About";

const App = () => (
  <Fragment>
    <BrowserRouter>
      <Switch>
        <Route
          exact
          path="/"
          render={(props) => (
            <Layout>
              <Home {...props} />
            </Layout>
          )}
        />
        <Route
          exact
          path="/about"
          render={(props) => (
            <Layout>
              <About {...props} />
            </Layout>
          )}
        />
        <Redirect to="/" />
      </Switch>
    </BrowserRouter>
  </Fragment>
);

export default App;
