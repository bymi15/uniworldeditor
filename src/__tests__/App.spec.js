import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('App', () => {
  test('should correctly render', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
