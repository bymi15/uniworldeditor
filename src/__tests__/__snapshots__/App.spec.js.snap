// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should correctly render 1`] = `ShallowWrapper {}`;
